plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2149,7 +2152,7 @@ So if getMinSpeed() returns null we fall back to getSpeed().
         /* Add a VirtIO channel for SystemVMs for communication and provisioning */
         if (vmTO.getType() != VirtualMachine.Type.User) {
             devices.addDevice(new ChannelDef(vmTO.getName() + ".vport", ChannelDef.ChannelType.UNIX,
-                              new File(_qemuSocketsPath + "/" + vmTO.getName() + ".agent")));
+                    new File(_qemuSocketsPath + "/" + vmTO.getName() + ".agent")));
```

Comment:
While you are busy with this, why not use File.seperator?

Reply:
@wido it's not me. It's IDEA did that occasionally.